keyboards/rainkeeb/rainkeeb.h

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2020 Regan Palmer
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT( \
+    K00, K10, K01, K11, K02,      K03, K13, K04, K14, K15, \
+    K20, K30, K21, K31, K22, K32, K23, K33, K24, K34, K25, \
+    K40, K50, K41, K51, K42, K52, K43, K53, K44, K54, K45, \
+    K60, K70, K61, K62,      K72,      K73, K64, K74, K65, \
```

Comment:
```suggestion
    K60, K70, K61, K62,      K72,      K73, K64, K74, K65  \
```

This comma broke the build....sorry about that.